pykern/pkcli/ci.py

```diff
@@ -24,7 +24,7 @@
     + r"|^run/"
 )
 _PRINT = re.compile(r"(?:\s|^)(?:pkdp|print)\(")
-_PRINT_OK = re.compile(r"^\s*#\s*(?:pkdp|print)\(")
+_PRINT_OK = re.compile(r"^\s*#.*(?:pkdp|print)\(")
```

Comment:
This is insufficient. A docstring with a print() does not work, and there were several that had to be fixed to pass check_prints.

The point really is that we are trying to be very strict about print statements. The case you pointed out didn't require a `print()`. If you want to avoid the check, you can always put a space between the `print` and `(`. However, I don't recommend that.